cmd/mqtt/main.go

```diff
@@ -62,9 +62,9 @@ const (
 	defThingsAuthTimeout = "1s"
 	envThingsAuthURL     = "MF_THINGS_AUTH_GRPC_URL"
 	envThingsAuthTimeout = "MF_THINGS_AUTH_GRPC_TIMEOUT"
-	// Nats
-	defNatsURL = "nats://localhost:4222"
-	envNatsURL = "MF_NATS_URL"
+	// Message broker
```

Comment:
Do we need this and other comments? Looks like it's the only service where it's done.

Reply:
This was initially done but I am changing it.